modules/installation/pages/server-setup.adoc

```diff
@@ -109,11 +109,26 @@ After synchronizing with SCC clients may receive updates, and be organized into
 
 This section covers synchronizing with SCC from the {webui} and adding your first client channel.
 
+Before you can synchronize software repositories with SCC, you will need enter Organization Credentials in {susemgr}.  These organization credentials (previously called mirror credentials) give you access to the SUSE product downloads.  You will find your organization credentials in https://scc.suse.com/organization.
+
+Enter your organization credentials in the {susemgr} {webui}:
+
+
+[[proc-admin-organization-credentials]]
+.Procedure: Entering Organization Credentials
+. In the {susemgr} {webui}, select menu:Main Menu[Admin > Setup Wizard].
+. From the [guimenu]``Setup Wizard`` page select the btn:[Organization Credentials] tab.
+. Click the btn:[Add a new credential].
+. In the pop-up, enter [guimenu]``Username`` and [guimenu]``Password``, and confirm with btn:[Save].
```

Comment:
Personally, I would use 'dialog' here, or write around it.

Reply:
yes, +1